definitions/fenix.toml

```diff
@@ -596,6 +596,22 @@ description = "Events Ping"
 build_id_column = "REPLACE(CAST(DATE(mozfun.norm.fenix_build_to_datetime(client_info.app_build)) AS STRING), '-', '')"
 
 
+[data_sources.events_stream]
+from_expression = """(
+    SELECT
+        p.*,
+        DATE(p.submission_timestamp) AS submission_date
+    FROM
+        `moz-fx-data-shared-prod.{dataset}.events_stream` p
+)"""
+client_id_column = "client_id"
+experiments_column_type = "glean"
+default_dataset = "org_mozilla_firefox"
```

Comment:
Should this just be `fenix`? I always get confused about the various fenix datasets..